examples/addons/websocket-inject-message.py

```diff
@@ -0,0 +1,31 @@
+"""
+Inject a WebSocket message into a running connection.
+
+This example shows how to inject a WebSocket message into a running connection.
+"""
+import asyncio
+
+from mitmproxy import ctx, http
+
+
+# Simple example: Inject a message as a response to an event
+
+def websocket_message(flow):
+    last_message = flow.websocket.messages[-1]
+    if b"secret" in last_message.content:
+        last_message.kill()
+        ctx.master.commands.call("inject", [flow], not last_message.from_client, "ssssssh")
```

Comment:
Can we hide this API and expose it as `flow.websocket.inject`? This would be much nicer for add-on authors.

Reply:
I see where you are coming from. `flow.websocket.inject` unfortunately is annoying for practical reasons. We need a reference to `flow`, which would introduce an unnecessary reference cycle between `HTTPFlow` and `WebSocketData`. We also need a reference to `master`, which we could get from the magic `ctx`, but again we are introducing a bit of spaghetti here.

I'd definitely be open to improve the API to call other commands. This really should be a first-class use case.